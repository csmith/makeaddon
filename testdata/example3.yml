package-as: SomeAddon

externals:
  WeakAuras/Libs/LibStub: https://repos.curseforge.com/wow/libstub/trunk
  WeakAuras/Libs/LibCustomGlow-1.0: https://github.com/Stanzilla/LibCustomGlow
  WeakAuras/Libs/LibSerialize:
    url: https://github.com/rossnichols/LibSerialize
    tag: v1.0.0

enable-nolib-creation: no

manual-changelog:
  filename: CHANGELOG.md
  markup-type: markdown

ignore:
  - README.md
  - CONTRIBUTING.md
  - update_translations.sh
  - generate_changelog.sh


move-folders:
  SomeAddon/SomeAddon: SomeAddon
  SomeAddon/SomeAddonTemplates: SomeAddonTemplates
  SomeAddon/SomeAddonArchive: SomeAddonArchive
