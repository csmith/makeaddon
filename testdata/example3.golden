{
  "PackageAs": "SomeAddon",
  "ManualChangelog": {
    "Filename": "CHANGELOG.md",
    "MarkupType": "markdown"
  },
  "Externals": {
    "WeakAuras/Libs/LibCustomGlow-1.0": {
      "Url": "https://github.com/Stanzilla/LibCustomGlow",
      "Tag": ""
    },
    "WeakAuras/Libs/LibSerialize": {
      "Url": "https://github.com/rossnichols/LibSerialize",
      "Tag": "v1.0.0"
    },
    "WeakAuras/Libs/LibStub": {
      "Url": "https://repos.curseforge.com/wow/libstub/trunk",
      "Tag": ""
    }
  },
  "MoveFolders": {
    "SomeAddon/SomeAddon": "SomeAddon",
    "SomeAddon/SomeAddonArchive": "SomeAddonArchive",
    "SomeAddon/SomeAddonTemplates": "SomeAddonTemplates"
  },
  "Ignore": [
    "README.md",
    "CONTRIBUTING.md",
    "update_translations.sh",
    "generate_changelog.sh"
  ],
  "LicenseOutput": "",
  "NoLibCreation": false
}